website/content/docs/nia/network-drivers/terraform-cloud.mdx

```diff
@@ -8,20 +8,19 @@ description: >-
 # Terraform Cloud Driver
 <EnterpriseAlert>
   This feature requires{' '}
-  <a href="https://www.hashicorp.com/products/consul/features">Consul-Terraform-Sync Enterprise</a>{' '}
+  <a href="https://www.hashicorp.com/products/consul/features">Consul-Terraform-Sync (CTS) Enterprise</a>{' '}
```

Comment:
I'll leave this up to you, but maybe the first `(CTS)` could be part of the first paragraph rather than this enterprise block?

Reply:
I wondered about this as well and was 50/50 on it. Sounds like it's worth changing to your suggestions then. Thanks!